src/app/core.ts

```diff
@@ -70,13 +70,26 @@ export const initDragDrop = (): Promise<boolean> => {
  * will be associated with this ID. It is recommended to use a unique ID to keep transfer
  * information private from other websites.
  *
+ * @param userSession whether or not to use multiple user sessions (defaults to false). This is
+ * useful to handle multiple user sessions in the same device.
+ *
  * @returns a promise that resolves if IBM Aspera Desktop is running properly or
  * rejects if unable to connect
  */
-export const initDesktop = (appId?: string): Promise<any> => {
+export const initDesktop = (appId?: string, userSession?: boolean): Promise<any> => {
+  if (asperaDesktop.globals.desktopVerified) {
+    return new Promise((_, reject) => {
+      reject('The Desktop SDK has already been initialized');
+    });
+  }
```

Comment:
```suggestion
  if (!asperaDesktop.isReady) {
    return throwError(messages.serverNotVerified);
  }
```

This is what we use elsewhere. Just create a new error message like `sdkAlreadyInitialized`.